executor/distsql.go

```diff
@@ -521,6 +525,10 @@ func (e *IndexLookUpExecutor) Close() error {
 	e.finished = nil
 	e.memTracker.Detach()
 	e.memTracker = nil
+	if e.runtimeStats != nil {
+		copStats := e.ctx.GetSessionVars().StmtCtx.RuntimeStatsColl.Get(e.idxPlans[0].ExplainID())
```

Comment:
Uh, so we can not record the table scan side of the index lookup reader?

Reply:
Because the table side scan rows are not so important.

Reply:
Uh, I think it sometimes can be useful, for example:
```
index lookup reader
|-> index scan
|
+-> selection
	|
	+-> table scan
```

with the row count of index scan and selection, we can check the selectivity estimated for selection.

Reply:
For this case, we can know the row count of `table scan` and `index lookup reader` in the output of `explain analyze`, so we can get the selectivity of this `selection`?

Reply:
Oh, row count of `table scan` is added to address this comment, got it.